src/ApiCompat/Rules/Compat/CannotMakeMoreVisible.cs

```diff
@@ -23,7 +23,7 @@ public override DifferenceType Diff(IDifferences differences, ITypeDefinitionMem
             // If implementation is protected then contract must be protected as well. 
             if (impl.Visibility == TypeMemberVisibility.Family)
```

Comment:
Oooh yes, I only cared about from impl to contract but the other way should also be included. Will fix that.